apps/eth/rpc.py

```diff
@@ -10,16 +10,17 @@
 
 class EventRPCClient(RPCClient):
 
-    def subscribe(self, project, url=Event.get_event_subscription_url(), interval=5000, events=None):
```

Comment:
It does not make sense to have a dict called `parameters` and then have 3 other parameters passed to the function after the dict.

Reply:
Remove the `parameters` dictionary and add `version` after `project`, make it a required parameter.